src/main/java/seedu/linkedout/MainApp.java

```diff
@@ -74,20 +74,20 @@ public void init() throws Exception {
      * or an empty linkedout book will be used instead if errors occur when reading {@code storage}'s linkedout book.
```

Comment:
Line 74: linkedout book comment does not make sense here, maybe change the comment wording.